TeaDiary.domain/Models/Tea.cs

```diff
@@ -4,8 +4,11 @@ namespace TeaDiary.domain.Models
 {
     public class Tea:Resource
     {
+        //public User User { get; set; }
```

Comment:
Why no navigation property?

Reply:
Cause didn't understand properly how it works. Learned it out. Fixed